src/index.js

```diff
@@ -110,13 +110,23 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
       }
 
       this.componentNode = findDOMNode(this.getInstance());
-      // return early so we dont initiate onClickOutside
+      // return early so we don't initiate onClickOutside
       if (this.props.disableOnClickOutside) return;
       this.enableOnClickOutside();
     }
 
-    componentDidUpdate() {
+    componentDidUpdate({ disableOnClickOutside: prevDisableOnClickOutside }) {
```

Comment:
This will break on IE11. As much as I want to burn IE to the ground, for now this code still has to live in a previous era of JS. We'll have to cache that value the older way.
```suggestion
    componentDidUpdate(prevProps) {
      var prevDisableOnClickOutside = prevProps.disableOnClickOutside;
```